routers/web/repo/compare.go

```diff
@@ -150,6 +150,9 @@ func setCsvCompareContext(ctx *context.Context) {
 		if err == errTooLarge {
 			return CsvDiffResult{nil, err.Error()}
 		}
+		if err != nil {
+			return CsvDiffResult{nil, err.Error()}
```

Comment:
At the moment, the Error is rendered into page directly, I am not sure whether there is any guarantee that the general error doesn't contain sensitive information (eg: server path).

Ideally, at the moment, I think the log could be:

```
if err != nil {
  log.Error("....", err)
  return CsvDiffResult{nil, errors.New("csv diff error")}
}
```

Reply:
You're totally right, I just checked for my use case and I did not think of this...